statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -8,6 +8,9 @@
 from __future__ import division, absolute_import, print_function
 from statsmodels.compat.python import long
 
+try: unicode
+except NameError: unicode = str
+
```

Comment:
@josef-pkt just wanted to check that this compatibility shim is fine as far as our general practices go

Reply:
I assume this is for py3 versus py2
It's better to add something like this to compat.python so we have it in a central location
There is an `asunicode` function there if that works for this.